### ISSUE TITLE
[Question] Can I dual boot Linux and Mac OS with asahilinux? Like switching between them when I want

### ISSUE THREAD

**Issue by Galorlx** [AsahiLinux/m1n1#176]

**Comment by rowanG077** [comment]
Yes you can choose between Linux and OSX in the bootpicker. See to the [FAQ](https://asahilinux.org/2022/03/asahi-linux-alpha-release/#can-i-dual-boot-macos-and-linux).

**Comment by Galorlx** [comment]
When would you guess Linux could support like full powerful usability

**Comment by Galorlx** [comment]
> Yes you can choose between Linux and OSX in the bootpicker. See to the [FAQ](https://asahilinux.org/2022/03/asahi-linux-alpha-release/#can-i-dual-boot-macos-and-linux).

When would you guess Linux could support like full powerful usability

**Comment by rowanG077** [comment]
I'm not the person to ask. But there is no official estimate, it's done when it's done. There are people working on userspace drivers.
